integration/integrate-using-generic-webhooks.adoc

```diff
@@ -0,0 +1,42 @@
+[id="integrate-using-generic-webhooks"]
+= Integrating by using generic webhooks
+include::modules/common-attributes.adoc[]
+:context: integrate-using-generic-webhooks
+
+toc::[]
+
+[role="_abstract"]
+With {product-title}, you can send alert notifications as JSON messages to any webhook receiver.
+When a violation occurs, {product-title} makes an HTTP POST request on the configured URL.
+The POST request body includes JSON-formatted information about the alert.
+
+The webhook POST request's JSON data includes a `v1.Alert` object and any custom fields that you configure.
+
+[source,json]
+----
+{
+  "alert": {
+    "id": "<id>",
+    "time": "<timestamp>",
+    "policy": {
+      "name": "<name>",
+      ...
+    },
+    ...
+  },
+  "<custom_field_1>": "<custom_value_1>"
+}
+----
+
+You can create multiple webhooks, for example, you can create a webhook for receiving all audit logs and a different webhook for alert notifications.
```

Comment:
This might read better as two sentences: 
> You can create multiple webhooks. For example...
also, `s/create a webhook for receiving all audit logs and a different webhook/create one webhook for receiving all audit logs and another webhook` _might_ sound better? Don't ask me to explain why! 😝